docs/fluttertodostutorial.md

```diff
@@ -1,483 +1,1488 @@
+
 # Flutter Todos Tutorial
 
 ![advanced](https://img.shields.io/badge/level-advanced-red.svg)
 
 > In the following tutorial, we're going to build a Todos App in Flutter using the Bloc library.
 
 ![demo](./assets/gifs/flutter_todos.gif)
+> ZZTODO: Update gif with the movie in the issue tracker.
 
 ## Key Topics
 
 - Observe state changes with [BlocObserver](/coreconcepts?id=blocobserver).
 - [BlocProvider](/flutterbloccoreconcepts?id=blocprovider), Flutter widget which provides a bloc to its children.
 - [BlocBuilder](/flutterbloccoreconcepts?id=blocbuilder), Flutter widget that handles building the widget in response to new states.
-- Using Bloc instead of Cubit. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
+- [BlocListener](/flutterbloccoreconcepts?id=bloclistener), a Flutter widget which invokes the listener code in response to state changes in the bloc.
+- Using 3 Blocs and  1 Cubit. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
+- Uses a [Repository and API layer](architecture/repository) separately for the data.
+- [RespositoryProvider](/flutterbloccoreconcepts?id=respositoryprovider), a flutter widget to provide a repository to its children.
 - Prevent unnecessary rebuilds with [Equatable](/faqs?id=when-to-use-equatable).
-- [MultiBlocProvider](/flutterbloccoreconcepts?id=multiblocprovider), a Flutter widget that merges multiple BlocProvider widgets into one.
+- [MultiBlocListener](/flutterbloccoreconcepts?id=multibloclistener), a Flutter widget that merges multiple BlocListener widgets into one.
```

Comment:
```suggestion
- [MultiBlocListener](/flutterbloccoreconcepts?id=multibloclistener), a Flutter widget that sets up multiple BlocListener widgets that allow us to react to different states.
```

Reply:
Not sure about this one, it can be reformed better

Reply:
Let's change to:
... sets up ...... at once. OR
... sets up ...... at the same point in the widget tree.

The "allows us to react to different states" isn't unique to MultiBlocListener. It's just a convenience /shortcut.

BTW: can you @marcossevilla add me as a collaborator on this issue or ask felix to do so? Otherwise, I can't suggest the change directly, I think, in github. 

Add suggestion to batch is greyed out.

Reply:
@chonghorizons you should be able to edit because it's your branch.

Reply:
@felangel Unfortunately, "Add suggestion to branch" is still greyed out:
![image](https://user-images.githubusercontent.com/36284839/146476453-34a2bc3f-400d-44d3-9157-20ae87202049.png)

I can edit the forked branch, but I think the issue is that I don't have write permission on this branch.